code/modules/economy/cash.dm

```diff
@@ -167,9 +167,39 @@
 /obj/item/spacecash/c1000/counterfeit
 	// If you're going to slap down huge sums of cash in easily reachable spots for RP or flavor reasons, make them counterfeit, so they don't break the black market - they will be rejected.
 	counterfeit = TRUE
+	
+/obj/item/spacecash/c10000
+	name = "10,000 dollars"
+	icon_state = "spacecash1000"
```

Comment:
you might just wanna make some sprites for these, given the sprites should be incredibly easy to just,,, make bigger